Src/partStream.cpp

```diff
@@ -225,9 +225,8 @@ main (int   argc,
     int nStreamPairs = locs.size();
     // Initialise particles
     Print() << "Initialising particles..." << std::endl;
-    for (int lev = 0; lev < Nlev; lev++) {
-      spc.InitParticles(locs,lev);
-    }
+    spc.InitParticles(locs,Nlev-1);
```

Comment:
Instead of passing Nlev-1 as an input, hardcode it in StreamPC.cpp and remove the argument